contents.json

```diff
@@ -5479,6 +5479,12 @@
       "category": "uitableview",
       "description": "A library to support collapsible sections in a table view.",
       "homepage": "https://github.com/jeantimex/CollapsibleTableSectionViewController"
+    }, {
+      "title": "Loaf",
+      "category": "ui",
```

Comment:
I think it will be better to move it to 'alert' category.

Reply:
Updated!